backend/src-common/src/main/java/org/eclipse/sw360/cyclonedx/CycloneDxBOMImporter.java

```diff
@@ -647,9 +647,13 @@ private Map<String, String> importAllComponentsAsPackages(Map<String, List<org.c
                             if (AddDocumentRequestStatus.DUPLICATE.equals(pkgAddSummary.getRequestStatus())) {
                                 Package dupPkg = packageDatabaseHandler.getPackageById(pkg.getId());
                                 if (!release.getId().equals(dupPkg.getReleaseId())) {
+                                    /* check if duPkg is orphan pkg or not */
+                                    if(!CommonUtils.isNotNullEmptyOrWhitespace(dupPkg.getReleaseId())){
```

Comment:
add this on line 649 in the same If condition separated by & operator. Also use CommonUtils.isNullEmptyOrWhitespace instead.